Verificación de Código Estático. Parte 1

JDGonzal opened the pull request:
npm i eslint eslint-config-airbnb-base eslint-plugin-import

Commit:
Verificación de Código Estático. Parte 1
npm i eslint eslint-config-airbnb-base eslint-plugin-import

aperdomob commented on package.json:
```diff
@@ -4,7 +4,8 @@
   "scripts": {
-    "test": "mocha -t 5000 --reporter mochawesome --reporter-options reportDir=report,reportFilename=ApiTesting"
+    "test": "npm run lint && mocha -t 5000 --reporter mochawesome --reporter-options reportDir=report,reportFilename=ApiTesting",
+		"lint": "eslint 'test/**/*.js'"
```
mala indentación

aperdomob commented on package.json:
```diff
@@ -19,6 +20,9 @@
     "mocha": "^3.5.3"
   },
   "dependencies": {
+    "eslint": "^4.8.0",
```
deben ir como dev-dependecies

aperdomob reviewed (changes requested):
Revisa esta regla que te falla en integración continua

![image](https://user-images.githubusercontent.com/17033942/31092511-4bb69c70-a774-11e7-9744-82b9aab0397d.png)

Commit:
Verificación de Código Estático. Parte 2
Corrección de Identación y texto en el package.json con "deben ir como dev-dependecies"
